src/monster.cpp

```diff
@@ -2236,6 +2253,15 @@ void monster::drop_items_on_death()
         items = remaining;
     }
 
+    // Adds all items in the monsters inventory to be dropped.
```

Comment:
This is already done in `monster::die` - https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/monster.cpp#L2120